src/Java.Interop.Tools.Maven/Repositories/IMavenRepository.cs

```diff
@@ -0,0 +1,12 @@
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using Java.Interop.Tools.Maven.Models;
+
+namespace Java.Interop.Tools.Maven.Repositories;
+
+public interface IMavenRepository
+{
+	// This name is used for on-disk caching purposes, and thus must be compatible with file system naming rules.
```

Comment:
Not sure if this means that directory names can also be included or not.  Is `foo/bar.xml` valid?

If directories names can't be included, perhaps this should be named `FileName`?

Reply:
Basically we need to place each Maven repository in its own directory in our cache to prevent collisions if they contain the same Java artifact.  In this example, it is "central" or "google":

```
- /MavenCacheDirectory
  -  /central/androidx.core/core/1.9.0/core-1.9.0.aar
  -  /google/androidx.core/core/1.9.0/core-1.9.0.aar
```  

I will try to describe it a little better in the comment.